src/main.tsx

```
@@ -24,7 +28,13 @@ if (import.meta.env.DEV) {
   }
 }
 
-createRoot(document.getElementById('root')!).render(
+const rootElement = document.getElementById('root');
```

ESLint

[error] Replace `'root'` with `"root"`

(prettier/prettier)